tests/acceptance/blockchain/conditions/test_conditions.py

```diff
@@ -490,7 +472,7 @@ def test_subscription_manager_get_policy_policy_struct_condition_key_context_var
         return_value_test=ReturnValueTest(
             comparator="!=",
             value=":sponsor",  # use sponsor sponsor context var
-            key=":sponsorIndex",
```

Comment:
L458 (the ":sponsorIndex" entry in the `context`) should also go away.

The test name itself, `test_subscription_manager_get_policy_policy_struct_condition_key_context_var_evaluation`, could probably use a rename since no longer testing `key_context_var`, perhaps `key_and_value_context_var`?